static/js/muting_ui.js

```diff
@@ -157,9 +157,9 @@ export function unmute_user(user_id) {
 }
 
 export function rerender_for_muted_user() {
-    message_lists.current.rerender();
+    message_lists.current.update_user_muting_and_rerender();
     if (message_lists.current !== message_lists.home) {
-        message_lists.home.rerender();
+        message_lists.home.update_user_muting_and_rerender();
     }
```

Comment:
This can be a follow-up, but we should probably do a refactor to make this pattern, which have in like 20 places, not involve duplication.  E.g. 
```
message_lists.for_all_lists((message_list) => message_list.update_muting_and_rerender());
```

@ryanreh99 FYI in case you have thoughts on the precise interface.